_includes/configuration/languages.md

```diff
@@ -4,10 +4,10 @@ The following languages are supported:
 
 | **Language**                                 | **Code**            |
 |----------------------------------------------|---------------------|
-| English (US)                                 | en, en-US           |
-| English (UK)                                 | en_uk, en_gb, en_br |
-| English (US) - with additional medical terms | en-US-medical       |
-| English (UK) - with additional medical terms | en-GB-medical       |
+| English (US)                                 | en, en_US           |
+| English (UK)                                 | en_GB, en_UK, en_BR |
+| English (US) - with additional medical terms | en_US-medical       |
```

Comment:
Is it worth a "requires 5.8" warning here? I know this doesn't fit within what we normally use that warning for, but I feel like we might want to show a "this didn't really work for a long time" caveat in the docs for this setting, and that's how we normally show those warnings elsewhere

Reply:
That would imply it's a feature added then, which wouldn't fit it's normal use case so I think we'd need a different warning if we wanted something here.

Reply:
It's also a tricky one, because up until a certain version of SpellChecker Pro it did work, so it more needs to be a certain version range which would be weird.

We also don't normally document things that were broken between certain version ranges like this, so my vote would be to not add anything here.

Reply:
technically, isn't it the spellchecker version, not tinymce version? the spellchecker version we are releasing should be (somewhat) backward compatible through 5.X?

Reply:
That is correct also :+1:

Reply:
verified internally. These are primarily docs fixes. Adding version information is not appropriate as the dictionaries were not added in this version, a bug prevented them from working as expected in previous versions.